templates/admin/http_stubs/includes/fieldset.html

```diff
@@ -16,7 +16,7 @@
                     {% else %}
                         {{ field.label_tag }}
                         {% if field.is_readonly %}
-                            <div class="readonly">{{ field.contents }}</div>
+                            {% if field.field.name == 'headers' %}<ul class="readonly">{% for header in field.contents|str_to_list:"', " %}<li>{{ header|cut:"'" }}</li>{% endfor %}</ul>{% else %}<div class="readonly">{{ field.contents }}</div>{% endif %}
```

Comment:
This approach looks extremely clumsy. With every change we just lay over one more layer of unreadable template code.

I'd suggest trying something else.

For the forms in general, django-crispy-forms is excellent.

And just for this case, for example, we may try using a separate field type or something.

But just adding ad-hoc conditions for every field will soon make the source harder to understand and maintain.

Reply:
Unfortunately, django-crispy-forms aren't not very good friends with simpleui, but if I used crispy I would make a template for this field.
I made custom change templates for every model. And set the attribute change_form_template in admin model.

Reply:
Well, unfortunately, I never worked with simpleui, and it's a long time since I've last worked with Django forms.
I need some time to experiment myself with Django built-in functionality and/or crispy.
Crispy forms allow to use any custom template for any field as far as I remember, although this might also look a bit clumsy.

Reply:
Decided to leave it as it is.